src/menu/settings_menu.c

```diff
@@ -90,6 +90,7 @@ typedef enum {
   VIEW_CARD_VERSION,
   VIEW_REGULATORY_INFO,
   PAIR_CARD,
+  CLEAR_USER_DATA,
```

Comment:
nitpicking. can we keep this just before the factory reset option?

Reply:
Updated!